### DISCUSSION TITLE
Wishlist

### DISCUSSION THREAD

**Discussion by nestoris** [nestoris/Win98SE#36] [Q&A]
What icons would you like to see and what changes to the theme do you think are needed?

**Comment by Dejweed** [comment]
Here's mine:

- [ ] Logout icons up to 96x96 (**system-log-out**, **system-reboot**, **system-shutdown**, **system-suspend**, **system-lock-screen**, **system-switch-user**, **system-suspend-hibernate**)
- [ ] Categories (mainly **applications-multimedia** and **applications-development**)
- [ ] Icon for Bittorrent clients (**transmission**)
- [x] Zoom actions (**zoom-in**, **zoom-out**...)
- [ ] **scanner**, **printer** 128x128
- [ ] **system-search**, **text-markdown**, **text-x-script** 160x160
- [ ] Disk burning (**brasero**)
- [ ] System tray (**xfce4-systray**)
- [ ] **panel**
- [ ] **edit-image**
- [ ] **window-pin**, **window-unpin**
- [ ] **mintwelcome**
- [ ] **mines**, **gnome-aisleriot**
- [ ] **object-locked**, **object-unlocked**
- [ ] **starred** 128
- [ ] **notification-disabled**
- [x] **preferences-system-power** 22 24
- [ ] **document-open-recent** 96

Bad scaling/not win98 style:
- [ ] **preferences-system-time** 64
- [ ] **authors** 64
- [ ] office 64 (**libreoffice-calc**,...)
- [ ] **git**
- [ ] **image-x-ico** 22 24 64

New icons:
- [x] **folder-games**
- [ ] **phone**
- [x] **view-right-new**, **view-left-close**, **view-right-close** (for dolphin)
- [ ] Color, icon and cursor theme settings (**preferences-color**, **preferences-desktop-icons**, **preferences-desktop-cursors**)
- [ ] Night color on/off (**redshift**)
- [ ] Icon for display brightness
- [ ] Icon for screenshot apps
- [ ] Icon for color picker apps
- [ ] Icon for firewall
